src/window/context_settings.rs

```diff
@@ -42,6 +42,29 @@ pub const CONTEXT_DEBUG: u32 = 1 << 2;
 #[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Debug, Copy)]
 pub struct ContextSettings(pub ffi::sfContextSettings);
 
+impl ContextSettings {
+    /// Creates a new, default ContextSettings.
+    pub fn new() -> ContextSettings {
```

Comment:
I'm okay with adding a new() method in addition to the `Default` implementation for consistency, but don't duplicate the initialization code. Use `Default::default()` here instead of copypasting the code.